IBPSA/Electrical/BaseClasses/PV/BaseClasses/PVOptical/package.mo

```diff
@@ -0,0 +1,19 @@
+within IBPSA.Electrical.BaseClasses.PV.BaseClasses;
+package PVOptical "Models for computing irradiance-related boundary conditions for PV systems"
+
```

Comment:
packages need to extend from `Modelica.Icons.Package` otherwise they are not rendered properly in OMEdit.

Reply:
done